src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.ts

```diff
@@ -19,6 +19,14 @@ export class InstructorHelpStudentsSectionComponent extends InstructorHelpSectio
     super();
   }
 
+  /**
+   * Returns a boolean value if any question in the subsection is to be displayed after the search
+   */
+  displaySubsection(array: Boolean[], firstPoint: number, lastPoint: number): boolean {
```

Comment:
move the function to after `ngOnInit()`?
and rename `array` to a more meaningful name like `toDisplaySubsection` or something?